app/assets/javascripts/diffusion_history/_map_utilities.es6

```diff
@@ -2,10 +2,6 @@ $(document).on("turbolinks:load", function () {
   google.maps.event.addDomListener(window, "load", initialize);
 });
 
-$(document).on("click", "#allMarkersButton", function (e) {
```

Comment:
⚠️ moved code specifically for diffusion map to `app/assets/javascripts/diffusion_history/home_map.es6`